rest/client/methods.go

```diff
@@ -741,37 +741,3 @@ func (c *communicatorImpl) ListHosts(ctx context.Context, td TaskData) ([]restmo
 	}
 	return hosts, nil
 }
-
-func (c *communicatorImpl) GetUserAuthorInfo(ctx context.Context, td TaskData, userID string) (*restmodel.APIUserAuthorInformation, error) {
```

Comment:
if we're removing this route, should we also remove it [on the server side](https://github.com/evergreen-ci/evergreen/blob/9f56a6caebf312020d3dd254af772caef0c32c30/rest/route/service.go#L76)?

Reply:
Yes thanks, didn't know about this!